demo/src/public/experiment-page2.js

```diff
@@ -1,14 +1,34 @@
 (function() {
     'use strict';
 
-    var lou = new Mozilla.TrafficCop({
+    var variants = {
+        'a': 30,
```

Comment:
I thought missing out the `v=` was a mistake at first, but I see it's deliberate. Definitely worth documenting?